fcrepo-exts/fcrepo-camel-toolbox
# FCREPO-3834 Enable Camel toolbox to send xml records to Solr indexing service

** FCREPO-3834 Enable Camel toolbox to send xml records to Solr indexing service **
* * *

**JIRA Ticket**: https://fedora-repository.atlassian.net/browse/FCREPO-3834

# What does this Pull Request do?
Removes the ldpath related code and replaces the Solr requirement on LDPath with a Solr XSLT transform implementation 

# What's new?
Replaced LDPath transform with an XSLT based transform which can be supplied per object via the same hasIndexingTransformation as the previous LDPath implementation. Alternatively, if no transofmation is specified, a bare-boines default transform is supplied on the classpathj which replicates the functaionality of the LDPath version.

# How should this be tested?

A description of what steps someone could take to:
* Make a call to the solr indexing endpoint and examing the resulting solr document whihc should be indexed on the configured solr core. 

# Additional Notes:README.md should probably be updated too to reflect the new changes and removal of the LDPath functionality, however, I wasn't sure whether that should wait until after a release as I believe the README.md is the main dicumentation for this repo and relates to the current release. 

# Interested parties
@fcrepo/committers
